alb.tf

```diff
@@ -1,154 +1,254 @@
+/**
+* https://github.com/terraform-aws-modules/terraform-aws-alb
+*/
+
 data "aws_lb" "public" {
-  count = var.alb_attach_public_target_group ? 1 : 0
-  name  = "public"
+  name = "public"
```

Comment:
does it makes sense to refactor this to a required variable as well instead of hardcoding this to `public`?

Reply:
unused.